massmigration/urls.py

```diff
@@ -5,7 +5,7 @@
 
 urlpatterns = [
     path("manage/", views.manage_migrations, name="massmigration_manage"),
-    path("run/<str:key>/", views.run_migration, name="massmigration_run"),
-    path("detail/<str:key>/", views.migration_detail, name="massmigration_detail"),
-    path("delete/<str:key>/", views.delete_migration, name="massmigration_delete"),
+    path("run/<str:key>/<str:db_alias>", views.run_migration, name="massmigration_run"),
+    path("detail/<str:key>/<str:db_alias>", views.migration_detail, name="massmigration_detail"),
+    path("delete/<str:key>/<str:db_alias>", views.delete_migration, name="massmigration_delete"),
```

Comment:
```suggestion
    path("run/<str:key>/<str:db_alias>/", views.run_migration, name="massmigration_run"),
    path("detail/<str:key>/<str:db_alias>/", views.migration_detail, name="massmigration_detail"),
    path("delete/<str:key>/<str:db_alias>/", views.delete_migration, name="massmigration_delete"),
```
Just keeping trailing slashes for consistency.